cm_generate_makefile.bash

```diff
@@ -59,6 +69,24 @@ get_kokkos_cuda_option_list() {
   done
 }
 
+get_kokkos_hip_option_list() {
+  echo parsing KOKKOS_HIP_OPTIONS=$KOKKOS_HIP_OPTIONS
+  KOKKOS_HIP_OPTION_CMD=
+  PARSE_HIP_LST=$(echo $KOKKOS_HIP_OPTIONS | tr "," "\n")
+  for HIP_ in $PARSE_HIP_LST
+  do
+     HIP_OPT_NAME=
+     if  [ "${HIP_}" == "rdc" ]; then
+        HIP_OPT_NAME=HIP_RELOCATABLE_DEVICE_CODE
+     else
+        echo "${HIP_} is not a valid hip option..."
+     fi
+     if [ "${HIP_OPT_NAME}" != "" ]; then
```

Comment:
What happens if I want to enable HIP without any particular options, i.e. just "-D Kokkos_ENABLE_HIP:BOOL=ON", will "${HIP_OPT_NAME}" be empty and `KOKKOS_HIP_OPTION_CMD` stay empty?

Reply:
@lucbv yes, it just stays empty and has no effect